dictionaries/en_US/src/en_US.txt

```diff
@@ -4552,6 +4554,7 @@ dashier
 dashpots
 dashy
 dataflow
+datacenter
```

Comment:
I would have moved it to software ducts

But why not

Reply:
Yes, this is more of a coding term.  `data center` is two words in standard English. Please add it as `dataCenter` to [`dictionaries/software-terms/src/coding-terms.txt`](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/coding-terms.txt).

Reply:
Is there a place where you documented the way to add words?

I'm talking about here the special case "dataCenter", sometimes there is a need to use `^` right?

I'm curious

Reply:
> Is there a place where you documented the way to add words?

You make a good point. I'm not sure if there is a good explanation. 
`
> I'm talking about here the special case "dataCenter"?

I suggest using `dataCenter` because normally it is two words. The software terms dictionary will generate `dataCenter` and `~datacenter`. `datacenter` gets used when the spell checking is not case sensitive.

Reply:
Why not asking to add `~datacenter` only?

I mean why is there a need for an uppercased `C`

I would love if you add a clear documentation.

It would help me to contribute. I have a lot of words from my personal projects too

Reply:
I opened

- https://github.com/streetsidesoftware/cspell-dicts/issues/3857

Reply:
So @virzak 

If we move discussion out, here you should add datacenter and datacenters to coding-terms dictionary

The syntax suggested is

dataCenter
dataCenters

As it helps to inform about the fact it's a compound words.